sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -26,6 +26,10 @@ internal class RadarActivityLifecycleCallbacks : Application.ActivityLifecycleCa
                     !ActivityCompat.shouldShowRequestPermissionRationale(activity, Manifest.permission.ACCESS_FINE_LOCATION)) {
                 RadarSettings.setPermissionsDenied(activity.applicationContext, true)
             }
+            if (ContextCompat.checkSelfPermission(activity.applicationContext, Manifest.permission.ACCESS_COARSE_LOCATION) == PackageManager.PERMISSION_DENIED &&
```

Comment:
This implementation is a bit of a "brute force".  It's the simplest change we can make, though I could see some potentially downside in terms of correctness, since we set `RadarSettings.setPermissionsDenied(activity.applicationContext, true)` if either fine OR coarse permissions have been denied.

Something I've been evaluating is the possibility of removing `RadarSettings.setPermissionsDenied(boolean)` altogether, and simply replacing calls to `RadarSettings.getPermissionsDenied()` with the calls happening here.

Another alternative is to create a second "permissions denied" setting.  So we could have one for fine and one for coarse.